lib/laa_crime_schemas/types/types.rb

```diff
@@ -109,18 +114,24 @@ module Types
     ].freeze
     IncomeBenefitType = String.enum(*INCOME_BENEFIT_TYPES)
 
-    OutgoingsType = String.enum(*%w[
-                                  rent
-                                  mortgage
-                                  board_and_lodging
-                                  council_tax
-                                  childcare
-                                  maintenance
-                                  legal_aid_contribution
-                                  self_assessment_tax_bill
-                                ])
-
-    EmploymentOutgoingsType = String.enum(*%w[self_assessment_tax_bill])
+    OTHER_OUTGOINGS_PAYMENT_TYPES = %w[
+      rent
+      mortgage
+      board_and_lodging
+      council_tax
+      childcare
+      maintenance
+      legal_aid_contribution
+    ].freeze
+    OtherOutgoingsPaymentType = String.enum(*OTHER_OUTGOINGS_PAYMENT_TYPES)
```

Comment:
You could actually separate housing payment types as well from this array, and then also concatenate the housing payment types array to make `OUTGOINGS_PAYMENT_TYPES` and reduce duplication a bit more?

Reply:
'none'  in HOUSING_PAYMENT_TYPES stopped me doing so